client/src/components/SignUp/SignUp.jsx

```diff
@@ -0,0 +1,82 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import { Card, Box, Typography, Button } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { useNavigate } from 'react-router-dom';
+import { signUpSchema } from './SignUpSchema';
+import ControllerTextField from './ControllerTextField';
+import SignUpError from './SignUpError';
+
+const SERVER_ADDRESS = 'http://localhost:3000';
```

Comment:
As discussed please put it into separate file and use to set axios default so we don't have to set it on per request basis

Reply:
done